docs/howto/krustlet-on-microk8s.md

```diff
@@ -0,0 +1,141 @@
+# Running Krustlet on [microk8s](https://microk8s.io)
+
+These are steps for running krustlet node(s) and [microk8s](https://microk8s.io) on the same machine.
+
+## Prerequisites
+
+You will require a running microk8s cluster for this guide. The steps below assume you will run microk8s and the Krustlet, on a single machine. `kubectl` is required but is installed with microk8s as `microk8s.kubectl`. The following instructions use `microk8s.kubectl` for simplicity. You may use a standlone `kubectl` if you prefer.
```

Comment:
Nit: Can you run a wrapper on this whole file? We generally wrap our MD at 100 chars

Reply:
Fixed fa4da36cf8aefb94f3025808d2b8d8da3dcd5bc1

Reply:
Fixed 2636da488bbad280e472e41552f6374e671c5be4